analyticsdataserver/settings/base.py

```diff
@@ -309,6 +312,12 @@
 # base url to generate link to user api
 LMS_USER_ACCOUNT_BASE_URL = None
 
+# storage settings for report downloads
+DEFAULT_FILE_STORAGE = 'django.core.files.storage.FileSystemStorage'
+MEDIA_ROOT = '/edx/app/analytics_api/analytics_api/static/reports'
+MEDIA_URL = 'http://localhost:8100/static/reports/'
```

Comment:
Since we are overriding the original MEDIA_URL definition in this file, it's not needed and can be deleted or replaced (on line 96).

Also, the port number would vary in different dev environments, but I can't think of a good way to get that in the settings or at runtime. So, for now devs can just override this in local.py if they run their dev server on a different port.